README.md

```diff
@@ -120,11 +126,24 @@ Usage
 
 Running sccache is like running ccache: wrap your compilation commands with it, like so:
 
-> $ sccache gcc -o foo.o -c foo.c
+```bash
+sccache gcc -o foo.o -c foo.c
+```
+
+If you want to use sccache for your rust builds you can define `build.rustc-wrapper` in the
+[cargo configuration file](https://doc.rust-lang.org/cargo/reference/config.html).  For example, you can set it globally
+in `$HOME/.cargo/config` by adding:
 
-or use it with rust, like so:
+```toml
+[build]
+rustc-wrapper = "[path to sccache]"
```

Comment:
I think the cargo warning should be addressed here, so that people are not surprised.  (And if there's a minimum version requirement, that'd be good to mention too.)